### PR TITLE
[furyosquad] Add website support

### LINKED ISSUE #311: Add FuryoSquad
http://www.furyosquad.com

### PR DESCRIPTION
**Pull request by XenonR**
Closes #311

### CODE REVIEW

File: src/web/mjs/connectors/FuryoSquad.mjs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class FuryoSquad extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'furyosquad';
+        super.label = 'Furyo Squad';
+        this.tags = [ 'manga', 'french', 'high-quality' ];
+        this.url = 'http://www.furyosquad.com/';
```
Comment (27136773) [inline src/web/mjs/connectors/FuryoSquad.mjs]: May remove trailing slash

File: src/web/mjs/connectors/FuryoSquad.mjs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    async _getMangas() {
+        const categories = ['/', '/en-cours', '/termines', '/stoppes'];
+
+        let request,data;
```
Comment (27136773) [inline src/web/mjs/connectors/FuryoSquad.mjs]: May declare variables when they are initialized (inside the loop).
No need to declare variables outside for re-use, there is no noticeable performance/memory gain.

File: src/web/mjs/connectors/FuryoSquad.mjs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                title: chapter.title.trim()
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, 'div.fs-reader-page source');
+
+        return data.map(page => page.src);
```
Comment (27136773) [inline src/web/mjs/connectors/FuryoSquad.mjs]: Is `page.src` always full qualified name (absolute) or should it be wrapped into `this.getAbsolutePath(page, this.url)`

**Review by 27136773 — APPROVED** [review]
Ready for merge

File: src/web/mjs/connectors/FuryoSquad.mjs (hunk 3 above)
Comment (XenonR) [reply src/web/mjs/connectors/FuryoSquad.mjs]: Uncertain, i missed that.

**Pull request merged**